### PR TITLE
Fixed htslib and DBI.pm install issues

### LINKED ISSUE #15: setup.sh failed again with htslib(probably)
Below is the error message.

---

ERRORS/WARNINGS FOUND IN PREREQUISITES.  You may wish to install the versions
of the modules indicated above before proceeding with this installation

Run 'Build installdeps' to install missing prerequisites.

Created MYMETA.yml and MYMETA.json
Creating new 'Build' script for 'Bio-DB-HTS' version '2.1'
Building Bio-DB-HTS
cc -I/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/htslib/htslib -I/usr/lib/x86_64-linux-gnu/perl/5.20/CORE -DVERSION="2.1" -DXS_VERSION="2.1" -fPIC -D_IOLIB=2 -D_FILE_OFFSET_BITS=64 -Wno-error -Wformat=0 -Wno-unused-result -c -D_REENTRANT -D_GNU_SOURCE -DDEBIAN -fwrapv -fno-strict-aliasing -pipe -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -O2 -g -o lib/Bio/DB/HTS/Faidx.o lib/Bio/DB/HTS/Faidx.c
ExtUtils::Mkbootstrap::Mkbootstrap('blib/arch/auto/Bio/DB/HTS/Faidx/Faidx.bs')
cc -shared -L/usr/local/lib -fstack-protector -o blib/arch/auto/Bio/DB/HTS/Faidx/Faidx.so lib/Bio/DB/HTS/Faidx.o -L/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/htslib -Wl,-rpath,/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/htslib -lhts -lpthread -lz
cc -I/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/htslib/htslib -I/usr/lib/x86_64-linux-gnu/perl/5.20/CORE -DXS_VERSION="2.1" -DVERSION="2.1" -fPIC -D_IOLIB=2 -D_FILE_OFFSET_BITS=64 -Wno-error -Wformat=0 -Wno-unused-result -c -D_REENTRANT -D_GNU_SOURCE -DDEBIAN -fwrapv -fno-strict-aliasing -pipe -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -O2 -g -o lib/Bio/DB/HTS.o lib/Bio/DB/HTS.c
ExtUtils::Mkbootstrap::Mkbootstrap('blib/arch/auto/Bio/DB/HTS/HTS.bs')
cc -shared -L/usr/local/lib -fstack-protector -o blib/arch/auto/Bio/DB/HTS/HTS.so lib/Bio/DB/HTS.o -L/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/htslib -Wl,-rpath,/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/htslib -lhts -lpthread -lz

Downloading required Ensembl API files
- fetching ensembl
  *\* GET https://github.com/Ensembl/ensembl/archive/release/84.zip ==> 302 Found (1s)
  *\* GET https://codeload.github.com/Ensembl/ensembl/zip/release/84 ==> 200 OK (2s)
- unpacking ./Bio/tmp/ensembl.zip
- moving files
- fetching ensembl-variation
  *\* GET https://github.com/Ensembl/ensembl-variation/archive/release/84.zip ==> 302 Found
  *\* GET https://codeload.github.com/Ensembl/ensembl-variation/zip/release/84 ==> 200 OK (2s)
- unpacking ./Bio/tmp/ensembl-variation.zip
- moving files
- fetching ensembl-funcgen
  *\* GET https://github.com/Ensembl/ensembl-funcgen/archive/release/84.zip ==> 302 Found (1s)
  *\* GET https://codeload.github.com/Ensembl/ensembl-funcgen/zip/release/84 ==> 200 OK (1s)
- unpacking ./Bio/tmp/ensembl-funcgen.zip
- moving files

Testing VEP script
/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/convert_cache.t ............. Can't locate DBI.pm in @INC (you may need to install the DBI module) (@INC contains: /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../ ./Bio /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.20.2 /usr/local/share/perl/5.20.2 /usr/lib/x86_64-linux-gnu/perl5/5.20 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.20 /usr/share/perl/5.20 /usr/local/lib/site_perl .) at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/DBSQL/DBConnection.pm line 74.
BEGIN failed--compilation aborted at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/DBSQL/DBConnection.pm line 74.
Compilation failed in require at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/DBSQL/DBAdaptor.pm line 68.
BEGIN failed--compilation aborted at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/DBSQL/DBAdaptor.pm line 68.
Compilation failed in require at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/Registry.pm line 136.
BEGIN failed--compilation aborted at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/Registry.pm line 136.
Compilation failed in require at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/Variation/Utils/VEP.pm line 82.
BEGIN failed--compilation aborted at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/Variation/Utils/VEP.pm line 82.
Compilation failed in require at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/convert_cache.t line 12.
BEGIN failed--compilation aborted at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/convert_cache.t line 12.
/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/convert_cache.t ............. Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/variant_effect_predictor.t .. Can't locate DBI.pm in @INC (you may need to install the DBI module) (@INC contains: /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../ ./Bio /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.20.2 /usr/local/share/perl/5.20.2 /usr/lib/x86_64-linux-gnu/perl5/5.20 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.20 /usr/share/perl/5.20 /usr/local/lib/site_perl .) at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/DBSQL/DBConnection.pm line 74.
BEGIN failed--compilation aborted at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/DBSQL/DBConnection.pm line 74.
Compilation failed in require at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/DBSQL/DBAdaptor.pm line 68.
BEGIN failed--compilation aborted at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/DBSQL/DBAdaptor.pm line 68.
Compilation failed in require at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/Registry.pm line 136.
BEGIN failed--compilation aborted at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/Registry.pm line 136.
Compilation failed in require at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/Variation/Utils/VEP.pm line 82.
BEGIN failed--compilation aborted at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/../Bio/EnsEMBL/Variation/Utils/VEP.pm line 82.
Compilation failed in require at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/variant_effect_predictor.t line 12.
BEGIN failed--compilation aborted at /srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/variant_effect_predictor.t line 12.
/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/variant_effect_predictor.t .. Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/filter_vep.t ................ ok    
## Test Summary Report

/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/convert_cache.t           (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
/srv/public/kjk/rainbow/bin/ensembl-tools-release-84/scripts/variant_effect_predictor/t/variant_effect_predictor.t (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
Files=3, Tests=29,  3 wallclock secs ( 0.03 usr  0.01 sys +  2.90 cusr  0.06 csys =  3.00 CPU)
Result: FAIL
Failed 2/3 test programs. 0/29 subtests failed.

### PR DESCRIPTION
**Pull request by soymintc**
to fix #15

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 # Created: December 23nd 2015
-# Last update: July 1st 2016
+# Last update: July 8th 2016
```
Comment (xenigmax) [inline setup.sh:3]: We will make a changelog.txt & git contains this information in detail.
So you don't need to do this.

File: setup.sh [hunk 2]
```diff
@@ -11,6 +11,12 @@ wkdir=`pwd` # working directory
 bin=$wkdir/bin
 [ ! -d $bin ] && { mkdir $bin; }
 
+pm=$HOME/perl5 # perl modules config
```
Comment (xenigmax) [inline setup.sh:14]: May I ask why? How can we sure about this?

File: setup.sh (hunk 1 above)
Comment (soymintc) [inline setup.sh:3]: I see. Should remove "Last update" lines?

File: setup.sh (hunk 2 above)
Comment (soymintc) [inline setup.sh:14]: BioPerl inside VEP automatically makes the $HOME/perl5 directory. 
All attempts (making perl_module directory within rainbow directory, making perl5 module within rainbow directory) without creating perl5 directory in $HOME failed.

File: setup.sh (hunk 1 above)
Comment (xenigmax) [inline setup.sh:3]: I think so.. but.. as you wish..

**Pull request merged**